### PR TITLE
Add new topology for smartswitch

### PR DESCRIPTION
**Pull request by ganglyu**
### Description of PR

Summary:
Fixes # (issue)
Microsoft ADO: 25068545

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
Need new topology to support smartswitch testbed

#### How did you do it?
Add new topology and related testbed information

#### How did you verify/test it?
Run add-topo and deploy-mg, check device configuration

### CODE REVIEW

**Comment by ganglyu** [comment]
> @ganglyu can we rename this topo as t1-smartswitch? Please refer a related PR in [sonic-net/sonic-buildimage#17326](https://github.com/sonic-net/sonic-buildimage/pull/17326)

I suggest using t1-8-lag-smartswitch, t1-32-lag-smartswitch, and t1-64-lag-smartswitch as topology names, please let me know what you think.

**Comment by Pterosaur** [comment]
Is it a dupliciated task with PR: https://github.com/sonic-net/sonic-mgmt/pull/14595 ?

**Comment by r12f** [comment]
seems to be the same work that Jing is doing? 

PR: https://github.com/sonic-net/sonic-mgmt/pull/14595 (Please ignore the demo folder).

File: ansible/vtestbed.yaml [hunk 1]
```diff
@@ -351,3 +351,20 @@
   inv_name: veos_vtb
   auto_recover: 'False'
   comment: Tests virtual switch vm
+
+- conf-name: vms-kvm-smartswitch
```
Comment (r12f) [inline ansible/vtestbed.yaml:355]: where is the topology update between NPU to DPU VMs?

**Comment by ganglyu** [comment]
> seems to be the same work that Jing is doing?
> 
> PR: #14595 (Please ignore the demo folder).

Need to check with Jing

File: ansible/vtestbed.yaml (hunk 1 above)
Comment (ganglyu) [reply ansible/vtestbed.yaml:355]: I only created VM for NPU and DPU in this PR.
We will have another PR to cover the connection between NPU to DPU VMs.

**Pull request closed without merging**